test/inference.jl

```diff
@@ -186,6 +186,7 @@ function find_tvar10930(arg)
 end
 @test find_tvar10930(Vararg{Int}) === 1
 
+const isleaftype = Base._isleaftype
```

Comment:
Couldn't this be written as `using Base: _isleaftype`? That's a little clearer, IMO.

Reply:
No; this file has a lot of uses of the function and I wanted to avoid the churn of changing all of them.

Reply:
Okay, thanks for the explanation